lib/components/tabs.vue

```diff
@@ -146,28 +142,28 @@
              * Move to next tab
              */
             nextTab() {
-                this.setTab(this.currentTab, false, 1);
+                this.setTab(this.currentTab + 1, false, 1);
             },
 
             /**
              * Move to previous tab
              */
             previousTab() {
-                this.setTab(this.currentTab, false, -1);
+                this.setTab(this.currentTab - 1, false, -1);
```

Comment:
I refactored how set tab works with the offset direction value (3rd argument to st tab), which now specifies a direction to move if the requested tab is disabled (skipping over disabled)